src/credentials/FabricTable.cpp

```diff
@@ -973,10 +1032,42 @@ CHIP_ERROR FabricTable::Init(PersistentStorageDelegate * storage)
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR FabricTable::Init(PersistentStorageDelegate * storage, OperationalKeystore * operationalKeystore)
+void FabricTable::Forget(FabricIndex fabricIndex)
 {
-    mOperationalKeystore = operationalKeystore;
-    return Init(storage);
+    VerifyOrReturn(IsValidFabricIndex(fabricIndex));
```

Comment:
This check seems a bit superfluous.

```suggestion
    VerifyOrReturn(IsValidFabricIndex(fabricIndex));
```

Reply:
Done